app/graphql/types/query_type.rb

```diff
@@ -44,26 +44,35 @@ def record_id(id:, index:)
 
       field :search, SearchType, null: false,
                                  description: 'Search for timdex records' do
-        argument :searchterm, String, required: false, default_value: nil
-        argument :citation, String, required: false, default_value: nil
-        argument :contributors, String, required: false, default_value: nil
-        argument :funding_information, String, required: false, default_value: nil
-        argument :identifiers, String, required: false, default_value: nil
-        argument :locations, String, required: false, default_value: nil
-        argument :subjects, String, required: false, default_value: nil
-        argument :title, String, required: false, default_value: nil
-        argument :from, String, required: false, default_value: '0'
+        argument :searchterm, String, required: false, default_value: nil, description: 'Query all searchable fields'
+        argument :citation, String, required: false, default_value: nil, description: 'Search by citation information'
```

Comment:
I sort of want to have a way to note arguments as `we don't recommend this as 99.999% of our data doesn't include this`. <sigh>

Reply:
I hear you. I think tutorial/explanation docs would be a good place to explain some of the limitations.